src/Delta/GithubMarkdown/Delta.php

```diff
@@ -22,8 +20,8 @@ abstract class Delta extends MarkdownDelta
      *
      * @return string
      */
-    protected function escape(string $insert): string
+    protected function escape($insert)
     {
-        return str_replace(['*', '#', '~'], ['\*', '\#', '\~'], $insert);
+        return str_replace(['*', '#', '~'], ['\\*', '\\#', '\\~'], $insert);
```

Comment:
This one still feels weird. Are we sure this doesn't have a different outcome of the method?

Reply:
The version produced by the library is actually the correct way of writing this. The intent is to have a \* in the resulting string, and it only works in the previous version because \* is an _invalid_ escape sequence. PHP should actually produce a warning for that, but it doesn't.

See also: https://stackoverflow.com/questions/3415683/in-php-do-i-need-to-escape-backslashes